fullstack-cert/html-projects/framed-videos/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css"/> 
+  <title>Display Videos in an iframe</title>
+</head>
+<body>
+  <h1>How to display videos in an iframe</h1>
+  <br>
```

Comment:
I believe this will be the first time `br` elements will be introduced. 
So I would suggest leaving a comment above as a remainder to teach it in the workshop.